src/injector.js

```diff
@@ -0,0 +1,14 @@
+import { Bottle } from 'bottlejs';
+
+
+export default function getInjector(scope = null) {
```

Comment:
I read the Bottle docs and they have their own global state. We don't need our own...

Reply:
and multiple scopes too